udata/core/dataset/api.py

```diff
@@ -327,17 +327,33 @@ def post(self):
 @ns.route("/recent.atom", endpoint="recent_datasets_atom_feed")
 class DatasetsAtomFeedAPI(API):
     @api.doc("recent_datasets_atom_feed")
+    @api.expect(dataset_parser.parser)
     def get(self):
+        args = dataset_parser.parse()
+        queryset = Dataset.objects.visible()
+        queryset = DatasetApiParser.parse_filters(queryset, args)
```

Comment:
```suggestion
        queryset = dataset_parser.parse_filters(queryset, args)
```
I think you can use the same parser as above